alembic/versions/c31b1a94a27d_initial_migration.py

```diff
@@ -0,0 +1,128 @@
+"""Initial migration
```

Comment:
One note about the initial migration was that I performed the exact steps described at https://github.com/magfest/ubersystem/tree/master/alembic but that led to me getting this error:

```
FAILED: Multiple heads are present; please specify the head revision on which the new revision should be based, or perform a merge.
```
I solved this by running ``sep alembic history`` in order to look up the current ``(uber) (head)`` and then specified that as the head for this revision by adding the ``--head 4947b38a18b1`` arguments to the autogenerate command.

I inspected the resulting file to confirm that it ended up looking right and compared it to other plugins' initial revisions.  I believe that this worked properly and did the correct thing.

Reply:
Yep, that was totally correct! If you wanted to be even more ambitious, you could update the README with instructions for creating a new plugin.

Reply:
I will do that!

Reply:
Although, I think you made it harder on yourself than it needed to be. Looking through alembic's documentation, I believe you could have done `--head uber@head` instead of looking up the actual revision hash.

Reply:
I found the same thing, but that didn't work for me.  I believe I just got the same error, but I'll scroll up and double-check before making a documentation PR.  (Maybe even try deleting the migration and re-generating it again using that argument to make sure.)

Reply:
One more thought! We could actually have our `sep alembic` wrapper detect if it's a new plugin, and use `uber@head` if `--head` isn't already specified. We're already making several similar conveniences.

Reply:
Good idea!  If I can get the ``--head uber@head`` to work then I'll do that, which will allow us to leave the docs untouched since it will make them retroactively correct - the second-best kind of correct!